packages/cli/__tests__/zostso/__system__/stop/cli.zos-tso.stop.address-space.system.test.ts

```diff
@@ -32,16 +32,18 @@ describe("zos-tso start address-space", () => {
 
     it("should throw an error if servlet key parameter is not provided", async () => {
         const response = runCliScript(__dirname + "/__scripts__/address-space/as_error_stop.sh", TEST_ENVIRONMENT);
-        expect(response.status).toBe(1);
         expect(response.stdout.toString()).toBe("");
-        expect(response.stderr.toString()).toMatchSnapshot();
+        expect(response.stderr.toString()).toContain('IZUG1126E: z/OSMF cannot correlate the request for ' +
+        'key "ZOSMFAD-55-aaakaaac" with an active z/OS application session');
+        expect(response.status).toBe(1);
     });
 
     it("should throw an error if provided address space is inactive", async () => {
         const response = runCliScript(__dirname + "/__scripts__/address-space/as_error_stop.sh", TEST_ENVIRONMENT);
-        expect(response.status).toBe(1);
         expect(response.stdout.toString()).toBe("");
-        expect(response.stderr.toString()).toMatchSnapshot();
+        expect(response.stderr.toString()).toContain('IZUG1126E: z/OSMF cannot correlate the request for ' +
```

Comment:
Should we flip the order to be 1st stderr and 2nd stdout (same question as above)?

Reply:
When we expect no error on stderr and meaningful output on stdout, I agree with that order.
When the test expects an error on stderr and nothing on stdout, I tend to want to see whatever is totally unexpected on stdout to be displayed before the test terminates in a failure.

Reply:
I kind of agree with the logic order as it is.
For a few number of tests (those expecting errors), we may want to know right away if there was some stdout.

That way we see a nice Jest output along the lines of:
```

Failed: ... should throw an error...  
...
"address space stopped successfully"
```